modules/message-lane/src/weights_ext.rs

```diff
@@ -36,6 +37,18 @@ pub fn ensure_weights_are_correct<W: WeightInfoExt>() {
 
 /// Extended weight info.
 pub trait WeightInfoExt: WeightInfo {
+	/// Returns weight of message send transaction (`send_message`).
+	fn send_message_overhead() -> Weight {
+		Self::send_minimal_message_worst_case()
+	}
+
+	/// Returns weight that needs to be accounted when message of given size is sent (`send_message`).
+	fn send_message_size_overhead(message_size: u32) -> Weight {
+		let message_size_in_kb = (1024u64 + message_size as u64) / 1024;
```

Comment:
Why do we add a kilobyte here? Shouldn't this just be `message_size as u64 / 1024`?

Reply:
We want to take a `ceil` of that divison, I guess we could in theory just add `1023` to be super precise, cause currently `1024 bytes` payload will be counted as `2kb`.